cloud-foundry/README.md

```diff
@@ -0,0 +1,37 @@
+# opa-policy
+[OPA](https://www.openpolicyagent.org/) bundles of Policies to be used with [conftest](https://github.com/instrumenta/conftest) for fitness function testing your platform
+
+Why you should care? Being able to test baseline values before attempting to deploy allows operators to *fail fast*. 
+
+These policies along with these tools are used to help bridge that gap.
+
```

Comment:
It'd be helpful if there were a bit more of an introduction to what this directory provides.  Maybe listing a couple of examples of the policies that are implemented.  Imagine someone knows about OPA and has maybe but not necessarily heard about conftest.  Just a few sentences, including examples of the policies in english that are in the directory.